arcee/api.py

```diff
@@ -33,17 +37,55 @@ def upload_qa_pairs(qa_set: str, qa_pairs: List[Dict[str, str]]) -> Dict[str, st
         Dict[str, str]: The response from the make_request call.
     """
     if len(qa_pairs) > 2000:
-        raise Exception("You can only upload 2000 QA pairs at a time")  
+        raise Exception("You can only upload 2000 QA pairs at a time")
 
     qa_list = []
     for qa in qa_pairs:
-        if "question" not in qa or "answer" not in qa:
+        if prompt_column not in qa.keys() or completion_column not in qa.keys():
             raise Exception("Each QA pair must have a 'question' and an 'answer' key")
 
-        qa_list.append({"question": qa["question"], "answer": qa["answer"]})
+        qa_list.append({"question": qa[prompt_column], "answer": qa[completion_column]})
 
     data = {"qa_set_name": qa_set, "qa_pairs": qa_list}
-    return make_request("post", Route.alignment+"/qaUpload", data)
+    return make_request("post", Route.alignment + "/qaUpload", data)
+
+def chunk_list(lst, chunk_size):
+    """Yield successive n-sized chunks from lst."""
+    for i in range(0, len(lst), chunk_size):
+        yield lst[i:i + chunk_size]
+
+def upload_instructions_from_csv(qa_set: str, csv_path: str, prompt_column: str = "prompt", completion_column: str = "completion", batch_size: int = 200) -> None:
+    """
+    Upload QA pairs from a CSV file to a specific QA set.
+
+    Args:
+        qa_set (str): The name of the QA set to upload to.
+        csv_path (str): The path to the CSV file containing QA pairs.
+
+    Returns:
+        Dict[str, str]: The response from the make_request call.
+    """
+    qa_pairs = []
+
+    print(f"Reading QA pairs from {csv_path}...")
+    with open(csv_path, 'r', encoding='utf-8') as csvfile:
+        reader = csv.DictReader(csvfile)
+        for row in reader:
+            if prompt_column not in row.keys() or completion_column not in row.keys():
+                #raise Exception(f"Each row must have a '{question_column}' and an '{answer_column}' key. You can override the column names using the question_column and answer_column arguments.")
+                raise Exception(f"Each row must have a '{prompt_column}' and an '{completion_column}' key. You can override the column names using the prompt_column and completion_column arguments.")
+            qa_pairs.append({
+                f"{prompt_column}": row[prompt_column],
+                f"{completion_column}": row[completion_column]
```

Comment:
Will this even work?  I thought we expected 

```
{
  "question": "..",
  "answer": ".."
}
```

Will it make it into the db?  

Any chance you could upload a CSV or a few CSVs you tested with to the PR?  Or add more details on how you tested the code

Reply:
Due to the remap in upload_qa_pairs, it still does